src/Logger.php

```diff
@@ -84,23 +79,30 @@ class Logger implements LoggerInterface
      */
     public function __construct(array $targets = [])
     {
+        $this->startTime = $_SERVER['REQUEST_TIME_FLOAT'];
```

Comment:
`$_SERVER['REQUEST_TIME_FLOAT']` is the same regardless of when it's called so it makes no sense to store it in a property.